cobigen-angular-client-generation/index.asciidoc

```diff
@@ -0,0 +1,30 @@
+= CobiGen Angular client generation
+====
+With the CobiGen code generator you can easily generate components for your angular application.
+The following tutorial provides an example for using the CobiGen cli.
+You can find more information on https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation
+====
+
+[step]
+--
+restoreDevonfwIde(["java", "mvn", "node", "ng", "npm"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you. The next step is to install the CobiGen CLI.
+[step]
+--
+installCobiGen()
+--
+
+CobiGen uses the backend classes for generating your code used in the angular application. So in the next steps you will create a java project, create a simple java class and run the backend server.
+[step]
+--
+createDevon4jProject("backend-server")
+--
+
+[step]
+== Build and run the Java backend server
+--
+buildJava("backend-server")
```

Comment:
I think the problem why the pipeline fails is the name of the project. The `-` in `backend-server` causes the error in the buildJava step.